packages/core/src/Stores/portfolio-store.js

```diff
@@ -472,7 +472,7 @@ export default class PortfolioStore extends BaseStore {
     }
 
     networkStatusChangeListener(is_online) {
-        this.is_loading = !is_online;
+        this.is_loading = this.is_loading || !is_online;
```

Comment:
@maryia-deriv Can you explain why we are making this change?  
Will it be set to false later if the network goes offline?

Reply:
@balakrishna-deriv  Yes, it will. We are making this change to make `this.is_loading` a priority, so if it's loading (e.g. fetching data), we should not change it to `false` just because the app went online. When the data fetch finishes, and `is_loading` is false, only then we want to consider `is_online` state. :)